mycoffee/functions.py

```diff
@@ -491,14 +495,18 @@ def get_result(params: Dict[str, Union[str, int, float]]) -> Dict[str, Union[str
     :param params: parameters
     """
     params_copy = params.copy()
+    params_copy["ratio"] = params_copy["coffee_ratio"] / params_copy["water_ratio"]
     params_copy["coffee"] = dict()
     params_copy["water"] = dict()
-    params_copy["coffee"]["cup"] = calc_coffee(params)
+    params_copy["coffee"]["cup"] = calc_coffee(
+        ratio=params_copy["ratio"],
+        water=params["water"],
+        water_unit=params["water_unit"],
+        coffee_unit=params["coffee_unit"])
     params_copy["coffee"]["total"] = params_copy["cups"] * params_copy["coffee"]["cup"]
     params_copy["water"]["cup"] = params["water"]
     params_copy["water"]["total"] = params_copy["cups"] * params_copy["water"]["cup"]
     params_copy["grind_type"] = get_grind_type(params_copy["grind"])
-    params_copy["ratio"] = round(params_copy["coffee_ratio"] / params_copy["water_ratio"], params["digits"])
     params_copy["strength"] = get_brew_strength(ratio=params_copy["ratio"])
```

Comment:
Here we removed the `round` function for calculating `ratio` I think that's the reason why some of the tests changed.